flygym/mujoco/arena/sensory_environment.py

```diff
@@ -22,7 +22,7 @@ class OdorArena(BaseArena):
     odor_source : np.ndarray
         The position of the odor source in (x, y, z) coordinates. The shape
         of the array is (n_sources, 3).
-    peak_intensity : np.ndarray
+    peak_odor_intensity : np.ndarray
```

Comment:
Can you rename the argument to `__init__` from `peak_intensity` to `peak_odor_intensity` too?